src/components/ip-checkbox/ip-checkbox.scss

```diff
@@ -0,0 +1,41 @@
+.checkbox {
+  position: relative;
+  cursor: pointer;
+  width: 18px;
+  height: 18px;
```

Comment:
open question to check with other devs 

Does such size should come from variable ? 
How should be controlled ?

@patrickIpedis @RomaricMitiana

Reply:
we can allow users to update size of checkbox manually in css via **--checkbox-size**
or create predefined size like 'small', 'medium', 'large' in params.